github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -559,7 +559,7 @@ presubmits:
       sriov-pod: "true"
     max_concurrency: 10
     name: pull-kubevirt-e2e-kind-1.22-sriov-nonroot
-    optional: true
+    optional: false
```

Comment:
nit: you can just remove the line, default is `false` for `optional`

Reply:
I wanted to be explicit because I always forget what the default is.